src/borg/remote.py

```diff
@@ -647,31 +647,31 @@ def handle_error(error, args):
 
     @api(since=parse_version('1.0.0'))
     def open(self, path, create=False, lock_wait=None, lock=True, exclusive=False, append_only=False):
-        pass
+        """actual remoting is done via self.call in the @api decorator"""
```

Comment:
hmm, not sure this is helpful - especially not if we generate api docs from that.